src/examples/nintendoswitch/debug-hello.go

```diff
@@ -0,0 +1,23 @@
+// This program is a minimal Nintendo Switch program that just outputs messages to
+// emulator console
+package main
```

Comment:
I think you may want to revise the path. Right now it builds as `tinygo build -target nintendo-switch examples/nintendoswitch`. You may want to change this to something like `examples/nintendoswitch/debug-hello`.

Reply:
That's true. I normalized everything to `nintendoswitch` now.